specification/columns/amortizationcategory.md

```diff
@@ -0,0 +1,51 @@
+# Amortization Category
+
+Amortization Category represents the various classifications of a charge
+based on it's eligibility for coverage by a commitment discount, and
+whether or not a commitment discount was actually applied.
+
+Amortization Category is used to identify and distinguish between types of charges
+that may require different handling, for example an upfront payment on
+a Savings Plan that is amortized over time into usage by other line items.
+
+The AmortizationCategory column adheres to the following requirements:
+
+* The AmortizationCategory column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) and MUST NOT be null.
+* This column is of type String and MUST be one of the allowed values.
+
+## Column ID
+
+AmortizationCategory
+
+## Display Name
+
+Amortization Category
+
+## Description
+
+Represents the various classifications of a charge
+based on it's eligibility for coverage by a commitment discount, and
+whether or not a commitment discount was actually applied.
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column type     | Dimension      |
+| Feature level   | Mandatory      |
+| Allows nulls    | False          |
+| Data type       | String         |
+| Value format    | Allowed values |
+
+Allowed values:
+
+| Value            | Description                          |
+| :--------------- | :------------------------------------|
+| Commitment Purchase | Original charge that is amortized over time into usage by other line items. |
+| Eligible - Covered | Usage that is discounted or covered by a commitment discount |
+| Eligible - Not Covered | Usage that could be covered by some form of commitment discount, but is not |
+| Not Applicable | Usage that is ineligible for commitment discounts of any kind. |
```

Comment:
Do we prefer "Not Applicable" or just plain null? Again, it's not about commitment discounts, but we could say amortization is not applicable. That feels like a placeholder value, which we say we don't support, tho 🤔
```suggestion
```

Reply:
To me it's not a placeholder, it's clearly delineating which line items are not able to be covered by a commitment discount (which I believe is where the eligibility conversation started).  I think `NULL` would do the job if we absolutely don't want to get into the eligibility topic.

Reply:
If you say "not applicable to be amortized", then I don't disagree. Whether we have this or not just depends on whether we want to prirotize being overly explicit or reduce data size when the value doesn't provide much value. I see both sides. I'm only barely on one side of that fence because I hear a lot of complaints about data size. I know customers of other cloud providers aren't as sensitive to that as my customers are, tho.